web/cypress/integration/plugin/create-delete-in-drawer-plugin.spec.js

```diff
@@ -32,7 +32,17 @@ context('Delete Plugin List with the Drawer', () => {
     cy.contains('Create').click();
 
     cy.contains(this.data.basicAuthPlugin).parents(this.domSelector.pluginCardBordered).within(() => {
-      cy.get('button').click({ force: true });
+      cy.get('button').click({
+        force: true
+      });
+    });
+
+    cy.get(this.domSelector.codeMirrorMode).invoke('text').then(text => {
+      if (text === 'UIForm') {
+        cy.get(this.domSelector.codeMirrorMode).click();
+        cy.get(this.domSelector.selectDropdown).should('be.visible');
+        cy.get(this.domSelector.selectDropdown + " [label=JSON]").click();
```

Comment:
it would be better to use string template here.
```suggestion
        cy.get(`${this.domSelector.selectDropdown} [label=JSON]`).click();
```

or define a new domselector would also be a good choice